recoil/backendUrlState.tsx

```diff
@@ -18,5 +18,5 @@ export const urls = [
 
 export const backendUrlState = atom<string>({
     key: "backendUrlState",
-    default: urls[0].url
+    default: urls[2].url
```

Comment:
```suggestion
    default: urls[0].url
```
Please keep the default URL.